scripts/start-utils

```diff
@@ -85,11 +81,7 @@ function isFalse() {
 }
 
 function isDebugging() {
-  if isTrue "${DEBUG:-false}"; then
-    return 0
-  else
-    return 1
-  fi
+  $(isTrue "${DEBUG:-false}")
```

Comment:
Should this just be the following, otherwise it'll try to execute the output of the isTrue operation
```suggestion
  isTrue "${DEBUG:-false}"
```